src/quicklist.c

```diff
@@ -1185,10 +1185,17 @@ int quicklistDelRange(quicklist *quicklist, const long start,
 
 /* compare between a two entries */
 int quicklistCompare(quicklistEntry* entry, unsigned char *p2, const size_t p2_len) {
-    if (unlikely(QL_NODE_IS_PLAIN(entry->node))) {
```

Comment:
Is `unlikely(QL_NODE_IS_PLAIN(entry->node)` mean `entry->value != NULL` ?

Reply:
No, the original code used `lpCompare` so we need to filter plain node, but now we use the value from `entry` which is already determined for us in `quicklistNext()`.